Src/BootCamp.Chapter/AccountDetails.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BootCamp.Chapter
+{
+    class AccountDetails
+    {
+        private readonly string _accountName;
+        private List<decimal> _accountBalanceHistory;
+        private decimal highestBalance;
+        private decimal biggestLoss;
+
+
+        public AccountDetails(string name)
+        {
+            _accountName = name;
+            _accountBalanceHistory = new List<decimal>();
+            highestBalance = 0;
+            biggestLoss = 0;
+        }
+
+        public string GetName()
+        {
+            return _accountName;
+        }
+
+        public void AddBalance(decimal balance)
+        {
+            _accountBalanceHistory.Add(balance);
+            highestBalance = Math.Max(balance, highestBalance);
+            biggestLoss = Math.Min(balance, biggestLoss);
```

Comment:
Biggest loss is calculated differently: 
You take a difference between ith and i +1 th balances and keep the result, then find the biggest such difference.